src/misc/sortByList.js

```diff
@@ -0,0 +1,36 @@
+const sortByList = [
+  {
+    name: "None",
+  },
+  {
+    name: "Alphabetically",
+  },
+  {
+    name: "Alphabetically (reversed)",
+  },
+  {
+    name: "By population (increasing)",
+  },
+  {
+    name: "By population (decreasing)",
+  },
+  {
+    name: "By regions",
+  },
+  {
+    name: "By area (increasing)",
+  },
+  {
+    name: "By area (decreasing)",
+  },
+];
+const sortBySwitchList = {
+  Alphabetically: "Alphabetically",
+  AlphabeticallyReversed: "Alphabetically (reversed)",
+  ByPopulationIncreasing: "By population (increasing)",
+  ByPopulationDecreasing: "By population (decreasing)",
+  ByRegions: "By regions",
+  ByAreaIncreasing: "By area (increasing)",
+  ByAreaDecreasing: "By area (decreasing)",
+};
+export { sortByList, sortBySwitchList };
```

Comment:
wykorzystujesz w zasadzie 2 razy te same dane - w przypadku, gdy trzeba będzie zmienic w jednym miejscu to następnie musiałbyś zmienić też w drugim - możesz zachować ten obiekt sortBySwitchList a same wartości wyciągnąć za pomocą 'Object.values(sortBySwitchList)' - dostaniesz tablicę taką samą jak w sortByList a w przypadku modyfikacji zmieniasz tylko w jednym miejscu :D

Reply:
nie będzie to obiekt z polem 'name' ale w zasadzie nie jest Ci potrzebne